lib/usd/ui/USDImportDialog.ui

```diff
@@ -23,17 +23,6 @@
    <bool>false</bool>
   </property>
```

Comment:
Qt Designer always seems to want to change the order that things show up here, which makes it hard to diff.  Changes made here are to add the Help menu item and then the two labels to display the counts.